### PR TITLE
5 get all articles

### PR DESCRIPTION
**Pull request by priscilla-02**
complete task 5 "get all articles"

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -56,3 +56,35 @@ describe("GET /api", () => {
...
+          expect(eachArticle).toHaveProperty("author");
+          expect(eachArticle).toHaveProperty("title");
+          expect(eachArticle).toHaveProperty("article_id");
+          expect(eachArticle).toHaveProperty("topic");
+          expect(eachArticle).toHaveProperty("created_at");
+          expect(eachArticle).toHaveProperty("votes");
+          expect(eachArticle).toHaveProperty("article_img_url");
+          expect(eachArticle).toHaveProperty("comment_count");
+          expect(article).not.toHaveProperty("body");
+        });
```
Comment (Keb84) [inline __tests__/app.test.js:81]: You should be testing to see if the articles come back in created_at order - descending. You could use the jest-sorted library for this in your test.

**Review by Keb84 — CHANGES_REQUESTED** [review]
Great work so far. Please see my comment to add a bit more to your tests and resubmit 👍

File: __tests__/app.test.js (hunk 1 above)
Comment (priscilla-02) [reply __tests__/app.test.js:81]: Added test with jest-sorted

**Review by johnhutcheon — APPROVED** [review]
As per our Slack message, Nick has approved this over Zoom, so good to merge!

**Pull request closed without merging**